src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,51 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
-
+        int buySum = 0;
+        int supplySum = 0;
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            String line = reader.readLine();
+            while (line != null) {
+                String[] lineValues = line.split(",");
+                if (lineValues[0].equals("buy")) {
+                    buySum += Integer.parseInt(lineValues[1]);
+                } else if (lineValues[0].equals("supply")) {
+                    supplySum += Integer.parseInt(lineValues[1]);
+                }
+                line = reader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+        int result = supplySum - buySum;
+        File toFileFile = new File(toFileName);
```

Comment:
The variable name `toFileFile` is a bit redundant. Consider renaming it to `outputFile` or `destinationFile` for clarity.